test.sh

```diff
@@ -5,21 +5,17 @@ set -e
 export LC_ALL=en_US.UTF-8
 
 TEST_ON_EL=$([ -f /etc/redhat-release ] && [ -x /usr/bin/yum ] && echo "true" || echo "false")
```

Comment:
Should this also become 
```suggestion
TEST_ON_EL=$([ -f /etc/redhat-release ] && [ -x /usr/bin/dnf ] && echo "true" || echo "false")
```

Reply:
Maybe? ;-)

I honestly do not remember what I thought back then in https://github.com/Katello/katello-certs-tools/commit/1398fa7c637f34454af66cb9939e23e20df593c9

Shouldn't `/etc/redhat-release` be sufficient?

Reply:
I think so too. Perhaps yum wasn't available on very bare containers?

Reply:
but then the tests wouldn't run anyway :shrug: